exp/rendering-architecture.md

```diff
@@ -0,0 +1,54 @@
+
+# Supported GPU drivers
+
+For NVIDIA GPUs, Anbox Cloud uses the [Enterprise Ready Driver (ERD) from NVIDIA](https://help.ubuntu.com/community/NvidiaDriversInstallation) for Linux as available in Ubuntu. 
+For AMD and Intel GPUs, Anbox Cloud uses the [Mesa driver](https://www.mesa3d.org/).
+
+See [Component versions](https://anbox-cloud.io/docs/component-versions) to refer to the actual version supported for any particular Anbox Cloud release.
+
+# Supported APIs
+
+For NVIDIA GPUs Anbox Cloud uses OpenGL ES 3.2 and EGL 1.5 graphical interfaces. For Intel and AMD GPUs, Anbox Cloud uses Vulkan 1.3.
+
+# Rendering architecture
+
+For NVIDIA GPUs, Anbox Cloud driver stacks currently have a native OpenGL driver. For AMD and Intel GPUs, an OpenGL or EGL driver is layered on top of the Vulkan API. Since Vulkan API provides better GPU management at a lower level than OpenGL or EGL, this approach is beneficial and preferred by many users.
+
+To have a better understanding of the rendering architecture of Anbox Cloud, it is important to understand what the Android framework offers in terms of rendering. In Android, applications interact with the SurfaceFlinger which is the system compositor that is responsible for composing a frame together from all the outputs rendered by different applications. The frame is then submitted to the hardware composer which renders the frames on a screen. For more information on Android graphics components and how they work, see https://source.android.com/docs/core/graphics. 
+
+# Rendering pipeline
+
+Anbox Cloud has two rendering pipeline models - one for NVIDIA and the other for Intel and AMD. However, irrespective of the GPU that you use, the path of a frame typically looks like this: Android application > SurfaceFlinger > Hardware composer > Anbox Cloud > Display on screen or send it to streaming component.
+
+For communication between the hardware composer module on the Android side and Anbox runtime, we use [Wayland](https://wayland.freedesktop.org/). So Anbox Cloud really functions as a compositor for Android i.e., the hardware composer module receives frames from the SurfaceFlinger and notifies Anbox runtime using Wayland. The Anbox runtime then submits the frame towards its output, which is either the screen or the streaming component.
+
+## For NVIDIA
+
+![Anbox Cloud NVIDIA pipeline|690x440](https://assets.ubuntu.com/v1/3ba1fddd-NVIDIA_pipeline.png)
+
+This NVIDIA pipeline uses the EmuGL stack. EmuGL is the hardware GLES emulation that is implemented with a mix of translator libraries, system libraries, and other rendering components. To understand more about EmuGL, see [EmuGL design overview](https://android.googlesource.com/platform/external/qemu/+/6654d90be28d0059baf854caccd4caf9f7033ccb/android/android-emugl/DESIGN). 
```

Comment:
We should not mention "EmuGL" or reference the Android docs but more describe the general concept of the API translation.

Reply:
Okay, we are already explaining in simple terms what exactly EmuGL does - take in API calls and do remote procedure calls to the driver. So I have removed the terminology and the link.